plugins/in_winevtlog/pack.c

```diff
@@ -381,6 +384,11 @@ static void pack_string_inserts(struct winevtlog_config *ctx, PEVT_VARIANT value
             msgpack_pack_str_body(ctx, "?", 1);
```

Comment:
It seems that I forgot to update this line and the one above to use `flb_log_event_encoder_append_body_cstring`, would you mind doing it?

Reply:
Done in [91c68db](https://github.com/fluent/fluent-bit/pull/7272/commits/91c68dbb558719a1275dcc673a658f015497368e)